client/stylesheets/_responsive.scss

```diff
@@ -298,6 +293,20 @@
 }
 
 @media (max-width: 650px) {
+    .profileCardInfo li:first-child {
+        font-size: 1rem !important;
+    }
+
+    .profileCardItem {
+        margin-bottom: 2px !important;
```

Comment:
!important should not be used<br>Properties should be ordered font-size, margin-bottom